lms/djangoapps/certificates/views/webview.py

```diff
@@ -665,6 +665,12 @@ def _render_invalid_certificate(course_id, platform_name, configuration):
 
 
 def _render_valid_certificate(request, context, custom_template=None):
+    certificate_data = context.get('certificate_data', {})
```

Comment:
I believe we should handle it in template while rendering the signatories here https://github.com/edx/edx-platform/blob/acf3cb684d3dd65c29699f04292e9a324bdb90e9/lms/templates/certificates/_accomplishment-rendering.html#L48-L58